src/main/scala/gitbucket/core/api/ApiRepository.scala

```diff
@@ -57,6 +63,7 @@ object ApiRepository {
       forks = 0,
       `private` = false,
       default_branch = "master",
-      owner = owner
+      owner = owner,
+      has_issues = false
```

Comment:
Super nit: Is there any reason why `has_issues` is `false` here? It's just a dummy data for web hook test, however, considering that `repository.issues.option` is default to `"PUBLIC"`, `true` might be better.

Reply:
yeah it is true. it is better to be `true` I fixed this too in https://github.com/gitbucket/gitbucket/pull/2542/commits/6339be6036d5ca81bad62f22929fcee61d8f99f3